### PR TITLE
Features/libsass

### PR DESCRIPTION
**Pull request by Snugug**
Move everything to partials to improve compile time

### CODE REVIEW

**Comment by Snugug** [comment]
Need to add a new reset in.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .vagrant/
 .www/
+.DS_Store
```
Comment (una) [inline .gitignore]: Can we put a global .gitignore in the virtual environment?

Comment (Snugug) [inline .gitignore]: No, because the git stuff is on the host machine not the guest.

Comment (una) [inline .gitignore]: won't we need to gitignore every instance of .DS_Store then? such as inside of individual folders?

Comment (Snugug) [inline .gitignore]: No. This covers all `.DS_Store` files.

**Pull request merged**